FinSearchUnified/tests/PluginTest.php

```diff
@@ -114,7 +113,8 @@ private function createTestProduct($number, $isActive)
         $article = $resource->create($testArticle);
 
         // Assertion to make sure the test article is created properly
-        $this->assertInstanceOf(\Shopware\Models\Article\Article::class, $article);
+        $this->assertInstanceOf(\Shopware\Models\Article\Article::class, $article,
+            'FINDOLOGIC' . $number . ($isActive ? ": active " : " inactive ") . " article created");
```

Comment:
- [ ] Rephrase the message since it will be shown when the assertion fails.

- [ ] Make sure to use `sprintf` if you want to have variable values in the message.

Reply:
should I remove this assertion instead? as it generally throws an `Exception` if for any reason the article is not created